test/tomo/operators/spect_trafo_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for the Single-photon emission computerized tomography transform."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+# External
+import pytest
+import numpy as np
+
+# Internal
+import odl
+from odl.tomo.operators.spect_trafo import AttenuatedRayTransform
+from odl.util.testutils import skip_if_no_niftyrec
+from odl.util.testutils import almost_equal
+
+
+@skip_if_no_niftyrec
+def test_spect_projector():
```

Comment:
This should be expanded:

* Test gpu
* Test attenuation
* Test PSF
* Test that the calculated values are correct in some sense (i.e. the correct "shape" according to angles and the correct "magnitude" according to our definition of the attenuated ray transform).